configurations/mt-staging.ts

```diff
@@ -115,6 +115,213 @@ const preEngagementConfig: PreEngagementConfig = {
+        {
+          value: "",
+          label: ""
+        },
+        {
+          value: "Unborn",
+          label: "Unborn"
+        },
+        {
+          value: "0",
+          label: "0"
+        },
+        {
+          value: "1",
+          label: "1"
+        },
+        {
+          value: "2",
+          label: "2"
+        },
+        {
+          value: "3",
+          label: "3"
+        },
+        {
+          value: "4",
+          label: "4"
+        },
+        {
+          value: "5",
+          label: "5"
+        },
+        {
+          value: "6",
+          label: "6"
+        },
+        {
+          value: "7",
+          label: "7"
+        },
+        {
+          value: "8",
+          label: "8"
+        },
+        {
+          value: "9",
+          label: "9"
+        },
+        {
+          value: "10",
+          label: "10"
+        },
+        {
+          value: "11",
+          label: "11"
+        },
+        {
+          value: "12",
+          label: "12"
+        },
+        {
+          value: "13",
+          label: "13"
+        },
+        {
+          value: "14",
+          label: "14"
+        },
+        {
+          value: "15",
+          label: "15"
+        },
+        {
+          value: "16",
+          label: "16"
+        },
+        {
+          value: "17",
+          label: "17"
+        },
+        {
+          value: "18",
+          label: "18"
+        },
+        {
+          value: "19",
+          label: "19"
+        },
+        {
+          value: "20",
+          label: "20"
+        },
+        {
+          value: "21",
+          label: "21"
+        },
+        {
+          value: "22",
+          label: "22"
+        },
+        {
+          value: "23",
+          label: "23"
+        },
+        {
+          value: "24",
+          label: "24"
+        },
+        {
+          value: "25",
+          label: "25"
+        },
+        {
+          value: ">25",
+          label: ">25"
+        },
+        {
+          value: "Unknown",
+          label: "Unknown"
+        },
+        {
+          value: "Other",
+          label: "Other"
+        }
+      ],
+    },
+    {
+      label: 'Select your gender/sess/Стать',
+      type: 'SelectItem',
+      attributes: {
+        name: 'gender',
+        required: true,
+        readOnly: false,
+      },
+      options: [
+        {
+          value: 'female',
+          label: '1. Female/Mara/Жінка',
+          selected: false,
+        },
+        {
+          value: 'male',
+          label: '2. Male/Raġel/Чоловік',
+          selected: false,
+        },
+        {
+          value: 'other',
+          label: '3. Others/Oħrajn/Інші',
+          selected: false,
+        },
+        {
+          value: 'notSay',
+          label: '4. Rather not say/Ma nixtieqx naghti risposta/Не хочу відповідати',
+          selected: false,
+        },
+      ],
+    },
+    {
+      label: 'How are you feeling/tħossok/почуття',
```

Comment:
this case is  tricky, lets leave it like that and check what the helpline thinks.